styles.css

```diff
@@ -730,6 +822,213 @@ body {
   cursor: pointer;
 }
 
+/* Estilos para la ventana emergente */
+
+/* Estilos Desktop Ocultos */
+.popup-desc-container-desktop {
+  display: none;
+}
+
+.popup-img-container-desktop {
+  display: none;
+}
+
+.popup-tech-buttons-desk {
+  display: none;
+}
+
+/* Estilos Desktop Ocultos */
+
+.projects-popup-window.show,
+.popup.show,
+.popup-description-container.show {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.popup-code.show {
+  display: flex;
+  position: fixed;
+  justify-content: center;
+  width: 100%;
+  z-index: 9999;
+  height: 100vh;
+  background-color: rgba(193, 199, 208, 0.8);
+  backdrop-filter: blur(0);
+  overflow-y: scroll;
+  align-items: center;
+}
+
+.popup {

```

Comment:
Great design team🤝

- [ ] [Optional] I would recommend you remove both the horizontal and vertical scrolls from your mobile and desktop popups. This will make your design a replica of the design given.